backend/chatsky_ui/api/api_v1/endpoints/flows.py

```diff
@@ -44,9 +53,17 @@ async def flows_get(
 
 @router.post("/")
 async def flows_post(
-    flows: Dict[str, Union[list, dict]], build_manager: BuildManager = Depends(get_build_manager)
+    flows: Dict[str, Union[str, Dict[str, Union[list, dict]]]], build_manager: BuildManager = Depends(get_build_manager)
 ) -> Dict[str, str]:
-    """Write the flows to the frontend_flows.yaml file."""
+    """Write the flows to the frontend_flows.yaml file.
```

Comment:
```suggestion
    """Write the flows to the frontend_flows.yaml file. Then commit changes to git without a tag.
```
tags are only created when building. their names are the builds ids

Reply:
Merge dev to your branch to see this fix